data/planet/outreachy/first-week-at-outreachy.md

```diff
@@ -0,0 +1,7 @@
+---
+title: First week experience at Outreachy
+description: To write down the experience of working at OCaml for outreachy internship
+url: https://medium.com/@aryangodara_19887/first-week-at-outreachy-54f4af71820f
+date: Tue Jun 13 16:36:50 IST 2023
+preview_image: https://miro.medium.com/v2/resize:fit:1400/format:webp/1*fLJmaO2fsscyePbR3Kgfkw.jpeg
+---
```

Comment:
You have to have some content there, it can't be empty

Reply:
I'm sorry I don't understand, Is the issue with the link to the image?